lexers/t/trafficscript.go

```diff
@@ -0,0 +1,19 @@
+package t
+
+import (
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+// TrafficScript lexer. For `Riverbed Stingray Traffic Manager
+// <http://www.riverbed.com/stingray>`
+var TrafficScript = internal.Register(MustNewLexer(
```

Comment:
It's named `Rts` in pygments. May we follow the same name to be ported as it is in pygments?
https://github.com/pygments/pygments/blob/a590ac5ea7c00a41e253834306bfa19e38349c0b/pygments%2Flexers%2Ftrafficscript.py#L20

Reply:
Forget about it.